pkg/executor/aggregate/agg_hash_executor.go

```diff
@@ -236,7 +236,11 @@ func (e *HashAggExec) Open(ctx context.Context) error {
 	if err := e.BaseExecutor.Open(ctx); err != nil {
 		return err
 	}
+	return e.OpenInternal()
+}
 
+// OpenInternal just opens the hash aggregation executor.
+func (e *HashAggExec) OpenInternal() error {
```

Comment:
Should we rename it to 'openInternal' to not expose this function since it's 'internal'?

Reply:
Renamed to "OpenExecutor" ? This is a part of logic from original Open function. The outside need to only open Agg executor so I separate it.

Reply:
Changed